### ISSUE TITLE
Feature: Segmentation losses/metrics

### ISSUE THREAD

**Issue by maju116** [maju116/platypus#75]
**Is your feature request related to a problem? Please describe.**
Add more losses/metrics for Semantic segmentation

**Additional context**
Check https://www.kaggle.com/bigironsphere/loss-function-library-keras-pytorch
